### DISCUSSION TITLE
Request for logging integration with Splunk using the Splunk HTTP Event Collector

### DISCUSSION THREAD

**Discussion by saad-littera** [vercel/community#620] [Idea]
Currently lacking integration with this important enterprise logging solution.

The current workaround would be:

vercel --> jason log draining --> aws firehose --> lambda function processing json --> splunk http event collector --> splunk cloud

**Comment by swarnava** [comment]
Hi there, Most of our integration are thirdparty and built by respective team. I will recommend reaching out to Splunk Support team to find out if they have something planned to support this: https://www.splunk.com/en_us/about-splunk/contact-us.html#customer-support. Thanks!
